src/js/components/GroupNavDMButtons/GroupNavDMButtons.js

```diff
@@ -83,8 +83,13 @@ class GroupNavDMButtons extends Component {
       <div className="nav-subgroup nav-subgroup-dm">
         <div className="nav-item nav-item-head">
           <div className="flex-grow">Direct Messages</div>
-          <div className="svg svg-plus add-group-dm" onClick={this.addButtonClickHandler} />
-          <div className="tool-tip">Create DM</div>
+          <div className="add-group-dm" onClick={this.addButtonClickHandler}>
```

Comment:
🤔 This makes me realise, that this should technically be something like a `button` or an `a` right?